src/routes/root.routes.js

```diff
@@ -5,36 +5,29 @@
 
 import Router from 'koa-router'; // @see https://github.com/alexmingoia/koa-router
 import {VERSION_PREFIX} from '../utils/version.util';
-import {CONFIG} from '../utils/config.util';
-
 import {renderFrontPage} from '../components/FrontPage/frontpage.component';
 
-import {wipeStores} from '../utils/test.util';
-
 const router = new Router({prefix: VERSION_PREFIX});
 
-router.get('/', (ctx, next) => {
+router.get('/', (ctx) => {
   ctx.body = renderFrontPage();
-  next();
 });
 
-router.get('/health', (ctx, next) => {
+router.get('/health', (ctx) => {
   ctx.body = 'OK!';
```

Comment:
Havde faktisk tænkt det samme. Opretter lige et issue på det, da man skal fange fejl på en anden måde i sanityCheck() for at det giver mening.